integration-tests/build.gradle

```diff
@@ -30,10 +34,34 @@ dependencies {
     testImplementation libraries.postgresql
     testImplementation libraries.mariadbJavaClient
     testImplementation libraries.springKafka
-    testImplementation libraries.testcontainers
     testImplementation libraries.awaitility
 }
 
+// junit4 deps are not used, exclude those
+configurations.testImplementation.exclude group: 'junit', module: 'junit'
```

Comment:
In some projects, I have used central registry for unwanted libs. For example in Webapp and in Payout: https://github.com/transferwise/payout-service/blob/master/build.common.gradle#L61

The benefit of that approach is, that it is in one place and prevents any kind of accidental transitive dependency coming in for any reason.

Reply:
Thanks, moved to build.common.gradle